lib/command/gherkin/steps.js

```diff
@@ -13,16 +13,22 @@ module.exports = function (genPath, options) {
 
   const codecept = new Codecept(config, {});
   codecept.init(testsPath, (err) => {
-    output.print('Gherkin Step definitions:');
-    output.print();
-    const steps = getSteps();
-    for (const step of Object.keys(steps)) {
-      output.print(`  ${output.colors.bold(step)} \n    => ${output.colors.green(steps[step].line || '')}`);
-    }
-    output.print();
-    if (!Object.keys(steps).length) {
-      output.error('No Gherkin steps defined');
-    }
+    if (err) throw new Error(`Error while running init :${err}`);
+
+    codecept.runBootstrap((err) => {
```

Comment:
bootstrap should not be run for listing steps definition (as it's not executed for def)